README.md

```diff
@@ -45,21 +46,63 @@ This is the dependency you need:
 </dependency>
 ```
 
-Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
-for more technical details.
+Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto) for more technical details. If you 
+have any question or suggestions, do not hesitate to create as issue or contact us in 
+[Telegram](https://t.me/artipie), our roadmap is available [here](https://github.com/orgs/artipie/projects/3/views/1).
```

Comment:
Think, `our roadmap` can be a separate sentence.
Shorter link `https://github.com/orgs/artipie/projects/3`.